CHANGELOG.txt

```diff
@@ -9,6 +9,18 @@
 
 ## 🩹 Patches
 
+- **1.10.3**: Pass on the information pane. This pane is still in development and there is much more to come.
+   - The info pane is now _much_ more friendly and easier to use.
+      - All metrics are always visible for all classes, but you will now see metrics that are "greyed out" if they're not applicable to that player's class.
+      - Back/Forward navigation added.
+      - Channel selection for reporting the metrics
+      - More to come!
+   - Significant performance improvements across the board
+   - Resolution of race conditions affecting Avoidable Damage tracking and damage events
+   - Substantial reduction in memory usage
+   - Streamlined code for better maintainability and future development
+   - Enhanced overall stability and reliability
+   - Bug fixes for various existing features
```

Comment:
Can you please make a suggestion I can commit, with much better documentation on this release in general? I wrote that early into the process, and a lot has changed since then.

Reply:
Can you outline some of the bug fixes directly? Also, remember delve support has been added, among other things. Keep the entire context in mind when building the changelog notes, and ensure you analyze all proposed code to convey all information that may be pertinent to an end-user. Particularly, memory leaks that used to cause the addon to lag quite badly, new features, enhancements, etc.